tests/integration/test_player_playoff_box_scores.py

```diff
@@ -119,3 +120,21 @@ def test_outputting_2019_playoff_box_scores_for_kawhi_leonard_as_csv(self):
                 expected_output_file.readlines(),
             )
         os.remove(output_file_path)
+
+    def test_get_playoff_box_scores_removes_dnp_games_by_default(self):
+        # Giannis missed one playoff game in 2020. Verify that the game from
+        # 9/8/2020 is not included in his boxscores.
+        results = playoff_player_box_scores(player_identifier="antetgi01", season_end_year=2020)
+        self.assertIsNotNone(results)
+        self.assertEqual(9, len(results))
```

Comment:
The other thing that might be good to verify (and I can do this after merge if you don't want to) is that the fields have `None` values.

Same for `tests/integration/test_player_regular_season_box_scores.py`.